### ISSUE TITLE
brick error: flag redefined: v

### ISSUE THREAD

**Issue by kroggen** [aergoio/aergo#161]
`brick` is showing an error when opened:

```
flag redefined: v
```

On [this line](https://github.com/aergoio/aergo/blob/develop/cmd/brick/brick.go#L90)

This error was not appearing in previous versions

**Comment by kroggen** [comment]
The output of `brick --help` shows that the `log` package (from aergo-lib) is using the `-v` flag

```
$ brick --help
Usage:
  brick [-p]
  brick [-p] [-o] [-w] <filename>
  -alsologtostderr
    	log to standard error as well as files
  -log_backtrace_at value
    	when logging hits line file:N, emit a stack trace
  -log_dir string
    	If non-empty, write log files in this directory
  -logtostderr
    	log to standard error instead of files
  -o	verbose output (only batch)
  -p	enable private features
  -stderrthreshold value
    	logs at or above this threshold go to stderr
  -v value
    	log level for V logs
  -vmodule value
    	comma-separated list of pattern=N settings for file-filtered logging
  -w	enable watch (only batch)
```

**Comment by kroggen** [comment]
@hayarobi 

Could you check this when you have available time? I saw that you worked on [aergo-lib/log](https://github.com/aergoio/aergo-lib/tree/master/log)

**Comment by kroggen** [comment]
This is already fixed on release/2.4.0
